pylint/checkers/exceptions.py

```diff
@@ -152,6 +159,11 @@ def _is_raising(body: typing.List) -> bool:
         "immediately. Remove the raise operator or the entire "
         "try-except-raise block!",
     ),
+    "W0707": (
+        "Exception needs to be raised with cause",
+        "raise-missing-from",
+        "Long explanation",
```

Comment:
```suggestion
        """Python 3's exception chaining means it shows the traceback of the current exception, but also the original exceptions. Not using raise from make the traceback inaccurate, because the message look like there is a bug in the exception-handling code itself, which is a separate situation than wrapping an exception.""",
```